### DISCUSSION TITLE
Error and unwanted logs when installing Python extension on vscode.dev

### DISCUSSION THREAD

**Discussion by chiaolinghong3d** [microsoft/pylance-release#6686] [Q&A] [answered]
First of all, I really appreciate the flexibility of using vscode on web through vscode.dev :)

**Steps to reproduce:**
1. Go on vscode.dev.
2. Open Extensions and install Python, click "Install Anyway" for the popup.
3. Open up a python file on my local machine.
4. Loads several errors in the console:
  [Extension Host] Unknown (FileSystemError) (FileSystemError): Unknown
  Refused to connect to 'blob: ...
  Uncaught (in promise) Error: Language client is not ready yet when handling queue/unleash
<img src="https://github.com/user-attachments/assets/5705574a-1601-4ae8-87f9-f199ccd9a6ed" width="750" />

I understand Python extension is not fully supported on the web, but is there a way to disable the logs from configurations or any plans on fixing the console errors? Thanks.

**Comment by karthiknadig** [comment]
This seems to be something from pylance. Moving it to the right repo.

**Comment by rchiodo** [comment, marked as answer]
As long as Pylance is working, I don't think we'd spend time trying to fix the console errors. The first one looks like a language file that's missing which is expected. It can be ignored.

The second one might be the callstack from that exact problem. It's a file not found error from VS code that could be from us trying to load a locale that doesn't exist.

The third one I'm not sure what it is but unless something isn't working, it shouldn't matter. 

The fourth one isn't ours. It's from the anycode extension.

**Reply by chiaolinghong3d** [reply]
Okay thank you so much :)
